src/test/java/io/rebloom/client/ClientTest.java

```diff
@@ -51,6 +50,29 @@ public void reserveAlreadyExists() {
         cl.createFilter("myBloom", 100, 0.1);
     }
 
+  @Test
+  public void reserveV2() {
+    cl.bfReserve("bfnonscale", 0.001, 2);
```

Comment:
Change the name to bfnormal?

Reply:
@ashtul Changed to `reserve-basic` some time ago.